Sources/HttpServer/Router/Router.swift

```diff
@@ -1,7 +1,6 @@
 public class Router {
     private var routesTable: RoutesTable
     private var responseBuilder: ResponseBuilder
-    let logger = Logger()
 
```

Comment:
Why are you removing the logger? I think in general, it's nice to have logs in applications, so you can get some more insight into what is going on with the code.